app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -15,16 +19,28 @@ import io.mockk.mockkStatic
 import io.mockk.slot
 import io.mockk.unmockkAll
 import io.mockk.verify
+import java.io.ByteArrayOutputStream
 import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.tasks.await
 
 class AuthRepositoryTest : ShouldSpec(
 	{
-		val firebaseAuth: FirebaseAuth = mockk(relaxUnitFun = true)
-		val tested = AuthRepository(firebaseAuth)
+		val quality = 50
 		val userPhoto = mockk<Uri>()
 		val userName = "Jack Smith"
 		val userEmail = "test@gmail.com"
+		val userUid = "test1234567890"
+		val byteArrayOutputStream = mockk<ByteArrayOutputStream>(relaxed = true)
+		val firebaseAuth: FirebaseAuth = mockk(relaxUnitFun = true)
+		val storageReference: StorageReference = mockk()
+		val bitmapConverter = mockk<BitmapConverter> {
+			every { convertBitmapToBytes(any(), any()) } returns byteArrayOutputStream
+		}
+		val firebaseStorage: FirebaseStorage = mockk(relaxed = true) {
+			every { reference.child("user/$userUid") } returns storageReference
+		}
+		val tested = AuthRepository(firebaseStorage, firebaseAuth, bitmapConverter)
+		isolationMode = IsolationMode.InstancePerTest
```

Comment:
i think we agreed on that test setup is on the beginning of the file, please check other tests

Reply:
Yes, a configuration in other places is at the beginning of the test. I will change it.